src/metldata/builtin_transformations/count_content_values/data_transform.py

```diff
@@ -85,14 +88,17 @@ def transform_class(
 
 def transform_resource(
     *,
-    referenced_resources: dict[ResourceId, Resource],
-    target_resource: Resource,
+    referenced_resources: dict[str, Any],
+    target_resource: dict,
     relation_name: str,
     instruction: CountContentValueInstruction,
 ):
     """Apply the count content value transformation to each resource of a class."""
-    target_content = target_resource.content
-    relation_target_ids = target_resource.relations.get(relation_name)
+    target_content = target_resource.get("content")
+    if not target_content:
+        raise EvitableTransformationError()
+
+    relation_target_ids = target_resource.get("relations", {}).get(relation_name)
```

Comment:
This changed behaviour in the same way as the line in `get_class_resources` and would benefit from the same change to distinguish both error sources.

Reply:
My reasoning is the same as described above.